### DISCUSSION TITLE
[Feature Request]: 手动添加种子和rss不识别下载

### DISCUSSION THREAD

**Discussion by ShiharaX** [jxxghp/MoviePilot#3396] [讨论]
### 当前程序版本

1.5.8

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

可不可以添加手动下载种子的选项和自定义rss订阅只下载不转移和识别的选项呀

**Comment by yangtao521** [comment]
遇到一样的问题，订阅他识别失败了 直接不下载
